packages/ui/src/validators/components/statistics/Era.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react'
+
+import { PercentageChart } from '@/common/components/charts/PercentageChart'
+import { BlockIcon } from '@/common/components/icons'
+import {
+  NumericValue,
+  StatisticItem,
+  StatisticItemSpacedContent,
+  StatisticLabel,
+  formatDurationDate,
+} from '@/common/components/statistics'
+import { DurationValue } from '@/common/components/typography/DurationValue'
+import { useStakingStatistics } from '@/validators/hooks/useStakingStatistics'
+export const Era = () => {
+  const { eraStartedOn, eraDuration, now, eraRewardPoints } = useStakingStatistics()
+  const { nextReward, percentage } = useMemo(() => {
+    const nextReward =
+      eraDuration && now && eraStartedOn ? Number(eraDuration) - (now.toNumber() - Number(eraStartedOn)) : undefined
```

Comment:
As I mentioned above `eraStartedOn` should become `undefined | someKindPolkadotOfUint` and also `eraDuration` is a constant so it can't be falsish so this code can be made a bit clearer:
```suggestion
    const nextReward = now && eraStartedOn && eraDuration - Number(now) - Number(eraStartedOn)
```

Reply:
I fixed the issue and pushed